src/components/smart-table-content/smart-table-content.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import { Table, Pagination, Button } from '../../';
+import TableHead from './table-head/table-head';
+import TableBody from './table-body/table-body';
+import styles from './smart-table-content.styles';
+import { colShape, sortingLocalizationShape } from './shapes';
+
+class SmartTableContent extends Component {
```

Comment:
Why not just "SmartTable" as name ?

Reply:
Because this is just about wrapping the content